### ISSUE TITLE
is there any max range?

### ISSUE THREAD

**Issue by Burle17** [sourpuh/ffxiv_waymarkstudio#2]
i try to make presets for farming (to mark the three farming spots with A B and C) but sometimes if i reload a preset for testing if it works it doesnt do all three waymarks just 2

only if i come closer and reload again it shows all three

so is there a max range of showing a waymark?

And if YES can we adjust this range?^^

**Comment by sourpuh** [comment]
Yes, there's a max range imposed by the game. As far as I'm aware there's no way to increase it.

I intend to add functionality to load presets when entering certain trigger areas. When that is available it could be used to load the preset when you're within range.

**Comment by Burle17** [comment]
ok i thought that already without wms i just never had to be far away because you farm and so you are close to the sport where you place the waymarks^^

your idea sounds great, i will gladly test it if it goes live at any time^^

**Comment by sourpuh** [comment]
Turns out I'm wrong and there isn't a max range; it was the safety checks I added that was preventing you from placing too far away. I added a new config option to disable the safety check so you can place from any range.
